<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QualiResultWidget</class>
 <widget class="QWidget" name="QualiResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="tbFilterVisible">
       <property name="text">
        <string>Filtern</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/view-filter.svg</normaloff>:/icons/icons/view-filter.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tbSaveToCsv">
       <property name="text">
        <string>CSV</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/text-csv.svg</normaloff>:/icons/icons/text-csv.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbSaveToPDF">
       <property name="text">
        <string>PDF</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/application-pdf.svg</normaloff>:/icons/icons/application-pdf.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2">
     <item>
      <widget class="QTableView" name="tvQualiResult">
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QDockWidget" name="dwFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>456</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>524287</height>
      </size>
     </property>
     <property name="floating">
      <bool>false</bool>
     </property>
     <property name="features">
      <set>QDockWidget::AllDockWidgetFeatures</set>
     </property>
     <property name="allowedAreas">
      <set>Qt::AllDockWidgetAreas</set>
     </property>
     <property name="windowTitle">
      <string>Filter</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="vlFilter">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Mitarbeiter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFilterEmployee">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Funktion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFilterFunc">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Schulung</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFilterTrain">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mitarbeitergruppe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFilterEmployeeGroup">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Ergebnisstatus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbTrainResultState">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string>Gut</string>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <item>
            <property name="text">
             <string>Gut</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ausreichend</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Schlecht</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Schulungsstatus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbTrainStateFilter">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>130</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="tbFilterReset">
             <property name="text">
              <string>Zur√ºcksetzen</string>
             </property>
             <property name="icon">
              <iconset resource="../icons.qrc">
               <normaloff>:/icons/icons/edit-reset.svg</normaloff>:/icons/icons/edit-reset.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbUpdate">
             <property name="text">
              <string>Aktualisieren</string>
             </property>
             <property name="icon">
              <iconset resource="../icons.qrc">
               <normaloff>:/icons/icons/update-none.svg</normaloff>:/icons/icons/update-none.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbSaveToPDF</sender>
   <signal>clicked()</signal>
   <receiver>QualiResultWidget</receiver>
   <slot>printToPDF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1021</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbSaveToCsv</sender>
   <signal>clicked()</signal>
   <receiver>QualiResultWidget</receiver>
   <slot>saveToCsv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>952</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFilterVisible</sender>
   <signal>clicked()</signal>
   <receiver>QualiResultWidget</receiver>
   <slot>switchFilterVisibility()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFilterReset</sender>
   <signal>clicked()</signal>
   <receiver>QualiResultWidget</receiver>
   <slot>resetFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>788</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbUpdate</sender>
   <signal>clicked()</signal>
   <receiver>QualiResultWidget</receiver>
   <slot>updateFilterAndCalculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>788</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dwFilter</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>QualiResultWidget</receiver>
   <slot>filterVisibilityChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFilterAndCalculate()</slot>
  <slot>switchFilterVisibility()</slot>
  <slot>printToPDF()</slot>
  <slot>resetFilter()</slot>
  <slot>saveToCsv()</slot>
  <slot>filterVisibilityChanged()</slot>
 </slots>
</ui>
